index.js

```diff
@@ -32,14 +31,14 @@ module.exports = function (source) {
     html: ${stringify(fm.html)},
     attributes: ${stringify(fm.attributes)}`;
 
-  if (!!options.vue && vueCompiler && vueCompilerStripWith) {
+  if (!!options.vue && vueCompilerStripWith) {
     const rootClass = options.vue.root || "frontmatter-markdown"
-    const compiled = vueCompiler.compile(`<div class="${rootClass}">${fm.html}</div>`)
+    const compiled = vueCompilerStripWith.compile(`<div class="${rootClass}">${fm.html}</div>`)
```

Comment:
ohhh, I didn't know what `vue-template-es2015-compiler` does exactly. thanks for getting rid of...!